File: src/main/java/org/ventanas/Java/Ejercicio7.java
package org.ventanas.Java;
// 7. Reverso de una cadena
// Pedir una cadena al usuario e imprimirla al revés.
import java.util.Scanner;

public class Ejercicio7 {
    public static void execute() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Inserte una cadena de caracteres: ");
        String cadena = scanner.nextLine();
        String reversed = new StringBuilder(cadena).reverse().toString();
        System.out.println(reversed);
    }
}

File: src/main/java/org/ventanas/Kotlin/Ejercicio7.kt
package org.ventanas.Kotlin
// 7. Reverso de una cadena
// Pedir una cadena al usuario e imprimirla al revés.

fun ejercicio7() {
    print("Inserte una cadena de caracteres: ")
    val cadena = readln().split("").reversed()
    cadena.forEach{
        print(it)
    }
    println()


}